.travis.yml

```diff
@@ -99,6 +99,8 @@ jobs:
     - make lint
     - make check-examples-webpack
     - if [ ${IS_HOME} = TRUE ]; then JOB=publish-examples buildtools/deploy.sh; fi
+    after_success:
+    - cat .build/coverage/lcov.info | node ./node_modules/coveralls/bin/coveralls.js
```

Comment:
There's the same code line 80, is it correct?

Reply:
Good question defectively, should I comment this line and we will uncomment it when we move on webpack?

Reply:
let's keep it